src/flamenco/runtime/sysvar/fd_sysvar_slot_history.c

```diff
@@ -120,7 +120,7 @@ fd_sysvar_slot_history_update( fd_exec_slot_ctx_t * slot_ctx ) {
 
 int
 fd_sysvar_slot_history_read( fd_exec_slot_ctx_t * slot_ctx,
```

Comment:
do we need the extra query + check here too?

Reply:
This is not in the cache